js/main.js

```diff
@@ -0,0 +1,99 @@
+$(document).ready(function() {
+ $.ajax({
+   url: 'http://localhost:3000/articles',
+   type: 'get'
+    dataType: 'json'
+ }).done(ArticleApp.getArticles);
+});
+
+// Create a namespace for my app
+ ArticleApp = ArticleApp || {};
+
+// callback handler for all articles from server
+ArticleApp.getArticles = function(data){
+ //debugger;
+ console.log(data);
+
+ var string = JSON.stringify(data);
+
+
+ // Iterate over all the articles returned from the
+ // server.
+
+ // And create html for each article
+
+ // The HTML for each article should be wrapped in a div
+ // that has and id "article-<id>"
+ // <div id="article-17"> .... </div>
+
+};
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+// $(document).ready(function() {
```

Comment:
An abundance of comments and unused code (without commentary/reason) isn't often a good thing, as it can confuse you and everyone on your team. If you have a section of commented code that's fine... but perhaps leave a comment about its purpose and such.